pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>1.1.0</Version>
+    <Version>1.1.1</Version>
```

Comment:
You will also need to update https://github.com/dotnet/core-setup/blob/release/1.1.0/build_projects/shared-build-targets-utils/DependencyVersions.cs to reflect the new servicing versions that come out of the CoreCLR 1.1.0 branch. I have kicked off https://devdiv.visualstudio.com/DevDiv/_build/explorer?_a=summary&buildId=435359 that should publish the packages whose version you should update with.